requirements/base.in

```diff
@@ -8,11 +8,15 @@ Sphinx
 sphinx-rtd-theme
 markdown
 
-Django<3.0
+Django~=3.2
 django-axes
 django-choices==1.7.0
 django-cors-middleware
+django-filter~=21.1
+django-utils-six
 django-markup==1.3
 django_redis
+django_capture_on_commit_callbacks
+
 djangorestframework
 vng-api-common<1.1.0
```

Comment:
Ik heb geprobeerd om met de nieuwe versies het project lokaal uit te proberen maar kreeg dit niet aan de praat (vanwege verouderde versies van bepaalde packages). `vng-api-common` zou ik iedergeval nog updaten als dat mogelijk is, daarnaast raad ik je aan om alle `.txt` bestanden uit de requirements folder te verwijderen en opnieuw de dependencies te genereren (en daarmee nieuwe `.txt` bestanden aan te maken).

Reply:
`pyyaml==3.13` [is bijvoorbeeld een versie uit 2018](https://pyyaml.org/wiki/PyYAML#history)